packages/cws-trx/src/config/types.ts

```diff
@@ -0,0 +1,113 @@
+import { transport } from '@coolwallet/core';
+
+export type Output = {
+  address: string,
+  value: number
+}
+
+export type Transport = transport.default;
+
+type SignTxData = {
+  transport: Transport,
+  appPrivateKey: string,
+  appId: string,
+  addressIndex: number,
+  publicKey: string | undefined,
+  confirmCB: () => void | undefined,
+  authorizedCB: () => void | undefined
+}
+
+type Transaction = {
+  // [key: string]: any,
+  refBlockBytes: string,
+  refBlockHash: string,
+  expiration: number,
+  timestamp: number
+}
+
+export interface NormalTradeData extends SignTxData {
+  transaction: NormalContract
+}
+
+export interface NormalContract extends Transaction {
+  contract: {
+    ownerAddress: string,
+    toAddress: string,
+    amount: number,
```

Comment:
`number` of transfer contract, `frozenBalance` of freeze contract, and `voteCount` of vote contract are all represented in sun unit, but we give them different types, `string` and `number`. How about we unify them to either `string` or `number` type.

Reply:
for any value that may be over 16 digits, we better wrap it as BigNumber to avoid the risk of arithmetic overflow, so these three parameters (amount, frozenBalance, voteCount) could be the acceptable types for the constructor of BigNumber, so I think "number" and "string of number" are both the acceptable types.